### ISSUE TITLE
LocalDate and LocalDateTime Axis types

### ISSUE THREAD

**Issue by thomasnield** [HanSolo/charts#6]
Hey Gerrit, 

There doesn't seem to be any good `LocalDate` and `LocalDateTime` axis types. Would this be something that's in scope for this library? 

I'm coming across this need a lot as I play with your charts and JavaFX charts in general.

Thomas Nield

**Comment by HanSolo** [comment]
Aloha,

Yep you are right there is nothing like that at the moment. I‘ve created one for the JavaFX charts some time ago and it should be possible to create one for the lib too.
Could you file an issue on github for the date axis?
Started with a new chart today which can be used to visualize two lines and an area between those lines where the fill color depends on which line is on top...darn not easy to describe...hereis an image :)

http://davidmerlemontgomery.blogspot.de/2009/09/two-color-xy-area-combo-chart.html?m=1

Cheers,

Gerrit

Gerrit Grunwald
Westfalenstr. 93
48165 Münster
Germany

tel: +49 (0)2501 24321
mob: +49 (0)171 1745350
web: http://www.harmonic-code.org


Am 1. Dez. 2017, 21:41 +0100 schrieb Thomas Nield <notifications@github.com>:

**Comment by thomasnield** [comment]
Yep just did. Haha I think you responded to the GitHub issue via email.

**Comment by HanSolo** [comment]
Darn...should read the mail first 😜

**Comment by thomasnield** [comment]
That makes perfect sense, I'll play with it. 

By the way, Edvin created a [tornadofx-charts](https://github.com/edvin/tornadofx-charts) repository, which will have the TornadoFX DSLS's around this library. I'll put the initial commit in later.

**Comment by thomasnield** [comment]
I have a project I need this for now. Lemme see if I can streamline this a little with a nice wrapper....
